build/tasks/publish-build-task.coffee

```diff
@@ -103,10 +103,17 @@ getAssets = ->
         assetName = "atom.#{arch}.rpm"
 
       cp sourcePath, path.join(buildDir, assetName)
+      assets = [{assetName, sourcePath}]
 
-      [
-        {assetName, sourcePath}
-      ]
+      # Check for an archive build on a debian build machine.
+      # We could provide a Fedora one if necessary and not compatible
```

Comment:
I replaced it with "we could provide a Fedora version if some libraries are not compatible"